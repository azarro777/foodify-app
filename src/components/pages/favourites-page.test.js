import React from 'react';
import FavouritesPage from './favourites-page';
import { shallow } from 'enzyme';

describe('Testing <FavouritesPage/> component', () => {
  const favourite = shallow(<FavouritesPage />);
  it('FavouritesPage have rendered correctly', () => {
    expect(favourite).toMatchSnapshot();
  });
});
