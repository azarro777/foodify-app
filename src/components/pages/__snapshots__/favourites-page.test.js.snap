// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <FavouritesPage/> component FavouritesPage have rendered correctly 1`] = `ShallowWrapper {}`;
